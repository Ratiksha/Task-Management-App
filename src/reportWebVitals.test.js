import { shallow } from 'enzyme';
import reportWebVitals from './reportWebVitals';

const wrapper = shallow(<reportWebVitals />);
it('should render reportWebVitals component', () => {   
  expect(wrapper).toMatchSnapshot();
});
