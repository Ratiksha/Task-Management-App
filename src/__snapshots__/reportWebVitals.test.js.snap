// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render reportWebVitals component 1`] = `ShallowWrapper {}`;
